nebu/xsl/cnxml-to-html5.xsl

```diff
@@ -286,7 +286,7 @@
 </xsl:template>
 
 <!-- Make an ugly exception for American Govt (https://trello.com/c/hXPF6dJa/) because it needs to be released soon and the full Pull Request that creates valid XHTML is too large to release quickly: https://github.com/Connexions/rhaptos.cnxmlutils/pull/157 -->
```

Comment:
Is the comment still valid?

Reply:
Good point! I removed the hacky template altogether (by removing the `|.//c:note` from the template above)